tests/explorer.api.tests/ApiTests.cs

```diff
@@ -193,6 +234,41 @@ public async Task FailWithInvalidApiKey()
                 });
         }
 
+        private async Task TestExploreResult(
+            HttpMethod method,
+            Guid explorerGuid,
+            TimeSpan pollFrequency,
+            ApiTestActionWithContent test,
+            [CallerMemberName] string vcrSessionName = "")
+        {
+            while (true)
+            {
+                using var response = await factory.SendExplorerApiRequest(method, $"/result/{explorerGuid}", null, nameof(ApiTests), vcrSessionName);
+                var content = await response.Content.ReadAsStringAsync();
+                using var jsonContent = JsonDocument.Parse(content);
+                var rootEl = jsonContent.RootElement;
+                Assert.True(
+                    rootEl.ValueKind == JsonValueKind.Object,
+                    $"Expected a JSON object in the response:\n{content}");
+
+                Assert.True(
+                    rootEl.TryGetProperty("status", out var status),
+                    $"Expected a 'status' property in:\n{content}");
+
+                Assert.True(
+                    rootEl.TryGetProperty("id", out var id),
+                    $"Expected an 'id' property in:\n{content}");
+                Assert.True(id.GetGuid() == explorerGuid);
+
+                test(response, content);
+                if (status.GetString() == "Complete")
+                {
+                    break;
+                }
+                System.Threading.Thread.Sleep(pollFrequency);
```

Comment:
This should be replaced with the `async` equivalent:

```suggestion
                await Task.Delay(pollFrequency);
```